examples/cord_epsim/Makefile

```diff
@@ -35,13 +33,25 @@ CFLAGS += -DDEVELHELP
 # For debugging and demonstration purposes, we limit the lifetime to the minimal
 # allowed value of 60s (see draft-ietf-core-resource-directory-11, Table 2)
 RD_LT ?= 60
-# Override this variable to set the RD server address (default is the all nodes
-# multicast address)
-RD_ADDR ?= \"ff02::1\"
-RD_PORT ?= 5683
-
 CFLAGS += -DCORD_LT=$(RD_LT)
-CFLAGS += -DCORD_SERVER_ADDR=$(RD_ADDR)
-CFLAGS += -DCORD_SERVER_PORT=$(RD_PORT)
+
+# If nothing else is defined, we use CoAP default port
+RD_PORT ?= COAP_PORT
+CFLAGS += -DRD_PORT=$(RD_PORT)
+
+# The RD server's address must be defined by the build environment by setting
+# the RD_ADDR environment variable. Per default, this value is set to the
+# loopback address for enabling the build tests to successfully build this
+# example.
+RD_ADDR ?= \"::1\"
+CFLAGS += -DRD_ADDR=$(RD_ADDR)
+
+# In case the default value for RD_ADDR is used, we display a warning to the
+# user.
+ifeq (\"::1\", $(RD_ADDR))
+  ifneq (clean, $(MAKECMDGOALS))
+    $(info WARNING: the RD server address is set to the loopback device!)
```

Comment:
Looks to me like it is this line that is causing problems on Murdock with its weird way to parse output, since `RD_ADDR` is set to `::1` above.